pkg/cmd/server/start/start_master.go

```diff
@@ -580,6 +582,29 @@ func checkForOverrideConfig(ac configapi.AdmissionConfig) (*overrideapi.ClusterR
 	return overrideConfig, nil
 }
 
+type GenericResourceInformer interface {
+	ForResource(resource schema.GroupVersionResource) (kinformers.GenericInformer, error)
+}
+
+type genericInformers struct {
```

Comment:
This was just the adapter interface for GC, if you want to start a brand new controller factory i want to do that in a separate PR where the rest of them get converted.